app/models/articles-model.js

```diff
@@ -0,0 +1,17 @@
+const db = require('../../db/connection')
+
+exports.selectArticleByID = (articleID)=>{
+    const queryStr = `SELECT * FROM articles WHERE article_id = $1`
+    return db.query(queryStr, [articleID])
+    .then(({rows})=>{
+        if(rows.length === 0){
+            return Promise.reject({
+                status: 404,
+                msg: "Not found"
+            })
+        } else{
+            return rows
```

Comment:
Here we want to be returning just an object, not an array with an object inside. Think about how we can just send the object back. Note: you will need to also update your test accordingly